RiotShareExtension/Model/ShareExtensionManager.m

```diff
@@ -49,94 +52,35 @@ + (instancetype)sharedManager
     static ShareExtensionManager *sharedInstance = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        
         sharedInstance = [[self alloc] init];
         
-        [[NSNotificationCenter defaultCenter] addObserver:sharedInstance selector:@selector(onMediaUploadProgress:) name:kMXMediaUploadProgressNotification object:nil];
-        
-        // Add observer to handle logout
```

Comment:
Why did you remove these observers?
We still have to handle correctly account logout or change.
I agree that we don't need to suspend/resume the session anymore, but `checkUserAccount` is still useful...